PrBuild/index.js

```diff
@@ -11,165 +11,61 @@ module.exports = function (context, req) {
         const sha = req.body.pull_request.head.sha;
         const target_branch = req.body.pull_request.base.ref;
         const pull_request = req.body.pull_request.id;
-        
-        const appcenter_token = process.env["APP_CENTER_TOKEN"];
-        const github_token = process.env["GITHUB_TOKEN"];
-        const config = require('./config.json');
-        const { repo_owner, repo_name, appcenter_owner, appcenter_app, branch_template, branch_regex, appcenter_owner_type } = config;
-
-        const repo_path = `${repo_owner}/${repo_name}`;
-
-        let message = "";
 
-        if(head_repo == repo_path) {
+        const appcenter_token = process.env['APP_CENTER_TOKEN'];
+        const github_token = process.env['GITHUB_TOKEN'];
 
-            if(action === "opened" || action === "synchronize") {
+        const config = require('./config.json');
+        const repo_config = config.repos.filter((repo) => {
+            return head_repo === `${repo.repo_owner}/${repo.repo_name}`;
+        });
+
+        if (repo_config.length === 1) {
```

Comment:
This doesn't look correct. One repository can be mapped to multiple App Center apps at the same time. We will want to give people the ability to trigger them in both places. For example:

React Native Repo A -> Android App Center Project A
React Native Repo A -> iOS App Center Project B

Turn this into a loop and process all the results in the filtered array.

Reply:
Got it, thanks for the clarification, iteration feature currently is in progress, will let you know when finish.